style.css

```diff
@@ -0,0 +1,80 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+html {
+  font-size: 10px;
+}
+
+body {
+  font-size: 1.6rem;
+}
+
+.container {
+  max-width: 1020px;
+  padding: 10px;
+  margin: auto;
+}
+
+.textarea {
+  width: 100%;
+  height: 20rem;
+  margin-bottom: 10px;
+
+}
+
+.keyboard {
+  border: solid 2px #666d89;
+  border-radius: 5px;
+}
+
+.keyrow {
+  display: flex;
+  justify-content: space-between;
+  padding: 5px;
+}
+
+.keyboard__key {
+  height: 50px;
+  min-width: 6%;
+  margin: 3px;
+  border: solid 2px #666d89;
+  border-radius: 5px;
+  color: #666d89;
+  font-size: 2rem;
+  box-shadow: 0 0 5px #666d89;
+  background-color: rgb(240, 240, 240);
```

Comment:
rgb(240, 240, 240) and  #666d89
how about one style for one file

Reply:
ok